README.md

```diff
@@ -273,8 +274,6 @@ If you've already cloned the repo without `--recurse-submodules`, you can do the
 git submodule update --init
 ```
 
-copying the schemas and test vectors into the protocol package's `resources` directory can be done by running `./gradlew syncSchemas` and `./gradlew syncTestVectors` respectively.
```

Comment:
is this no longer relevant?

Reply:
correct @KendallWeihe - schemas and test vectors are now coming in via `tbdex` submodule, so all we have to do is the command above to keep in sync, or push changes if we make changes to the schema or test vectors here.